CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 ## 0.7.1
+
```

Comment:
oh, just for my understanding, is the space for formatting or for npm to parse release logs properly?

Reply:
Oh I don't think it matters, my VS Code autoformatter just added these.